.github/workflows/stack.yaml

```diff
@@ -0,0 +1,67 @@
+name: "Full-stack tests"
+
+on: push
```

Comment:
Probably also daily and on manual trigger?

Reply:
Good call!